scripts/config.ts

```diff
@@ -378,7 +378,7 @@ function writeL3ChainConfig(argv: any) {
             "AllowDebugPrecompiles": true,
             "DataAvailabilityCommittee": false,
             "InitialArbOSVersion": 30,
-            "InitialChainOwner": "0x0000000000000000000000000000000000000000",
+            "InitialChainOwner": argv.l2owner,
```

Comment:
wonder if this changing the rollup creation salt and thus address would cause issue downstream

Reply:
not 100% sure, but I think in most places/repos we moved away from hard-coded  addresses and instead get them by parsing files